setup.cfg

```diff
@@ -83,22 +83,27 @@ open_files_ignore = test.fits asdf.fits
 # which pytest trips over during collection:
 filterwarnings =
     ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes
+    ignore:numpy.ndarray size changed:astropy.utils.exceptions.AstropyWarning
+    ignore:numpy.ndarray size changed:RuntimeWarning
 # Configuration for pytest-doctestplus
 text_file_format = rst
 # Account for both the astropy test runner case and the native pytest case
 asdf_schema_root = asdf-standard/schemas asdf/schemas
-asdf_schema_skip_names =
-    asdf-schema-1.0.0
-    draft-01
-    celestial_frame-1.0.0
-    celestial_frame-1.1.0
-    frame-1.1.0
-    spectral_frame-1.1.0
-    step-1.1.0
-    step-1.2.0
-    wcs-1.1.0
-    wcs-1.2.0
-asdf_schema_skip_examples = domain-1.0.0 frame-1.0.0
+asdf_schema_skip_tests =
+    stsci.edu/asdf/asdf-schema-1.0.0.yaml
+    stsci.edu/asdf/transform/domain-1.0.0.yaml
+    stsci.edu/asdf/wcs/celestial_frame-1.0.0.yaml
+    stsci.edu/asdf/wcs/celestial_frame-1.1.0.yaml
+    stsci.edu/asdf/wcs/frame-1.0.0.yaml
+    stsci.edu/asdf/wcs/frame-1.1.0.yaml
+    stsci.edu/asdf/wcs/spectral_frame-1.1.0.yaml
+    stsci.edu/asdf/wcs/step-1.1.0.yaml
+    stsci.edu/asdf/wcs/step-1.2.0.yaml
+    stsci.edu/asdf/wcs/wcs-1.1.0.yaml
+    stsci.edu/asdf/wcs/wcs-1.2.0.yaml
+    stsci.edu/yaml-schema/draft-01.yaml
+asdf_schema_xfail_tests =
+    stsci.edu/asdf/core/ndarray-1.0.0.yaml::test_example_2
```

Comment:
Is there a reason to change the config name from `asdf_schema_skip_names` to `asdf_schema_skip_tests`?  I suspect so that the full path can be used and not break existing usage?

Reply:
Yeah, exactly.  I didn't like using only the schema basename but also didn't want to break anything.